modules/distribution/src/bin/fips.bat

```diff
@@ -16,11 +16,21 @@ rem KIND, either express or implied.  See the License for the
 rem specific language governing permissions and limitations
 rem under the License.
 
-set BC_FIPS_VERSION=1.0.2.4
-set BCPKIX_FIPS_VERSION=1.0.7
+set BC_FIPS_VERSION=2.1.0
+set BCPKIX_FIPS_VERSION=2.1.9
+set BCUTIL_FIPS_VERSION=2.1.4
 
-set EXPECTED_BC_FIPS_CHECKSUM=da62b32cb72591f5b4d322e6ab0ce7de3247b534
-set EXPECTED_BCPKIX_FIPS_CHECKSUM=fe07959721cfa2156be9722ba20fdfee2b5441b0
+set EXPECTED_BC_FIPS_CHECKSUM=c8df3d47f9854f3e9ca57e9fc862da18c9381fa9
+set EXPECTED_BCPKIX_FIPS_CHECKSUM=722eaefa83fd8c53e1fc019bde25e353258ed22b
+set EXPECTED_BCUTIL_FIPS_CHECKSUM=1d37b7a28560684f5b8e4fd65478c9130d4015d0
```

Comment:
what do you think of refactor into a helper function?

install_fips() {
  module=$1 version=$2 checksum=$3
  …
}
install_fips bc-fips   $BC_FIPS_VERSION   $EXPECTED_BC_FIPS_CHECKSUM
install_fips bcpkix-fips $BCPKIX_FIPS_VERSION $EXPECTED_BCPKIX_FIPS_CHECKSUM
install_fips bcutil-fips $BCUTIL_FIPS_VERSION $EXPECTED_BCUTIL_FIPS_CHECKSUM

Reply:
This script requires a lot of refactoring, we can handle it in a separate PR